packages/common/src/index.test.ts

```diff
@@ -25,9 +25,9 @@ describe('Common index', () => {
             expect(formatValue('km', undefined, '5')).toEqual('5 km');
 
             expect(formatValue('mi', undefined, 5)).toEqual('5 mi');
-            expect(formatValue('usd', undefined, 5)).toEqual('$5');
-            expect(formatValue('gbp', undefined, 5)).toEqual('£5');
-            expect(formatValue('eur', undefined, 5)).toEqual('€5');
+            expect(formatValue('usd', undefined, 5)).toEqual('$5.00');
```

Comment:
I'm not sure if is a good idea to always assume "round 2" when using currencies by default. /cc @TuringLovesDeathMetal  ?

Reply:
Agree, Thought about it, As per MDN, if the value is not provided and not in [ISO currency code list](https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list-one.xml), it defaults to 2. 
So if the round is not provided, it would follow code list ( [Ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#:~:text=the%20default%20for%20currency%20formatting%20is%20the%20number%20of%20minor%20unit%20digits%20provided%20by%20the%20ISO%204217%20currency%20code%20list%20(2%20if%20the%20list%20doesn%27t%20provide%20that%20information).) )

Reply:
^great suggestion @muhsinkamil 

If round is not specified, then we give them a good default (i.e. the default from the code list). 

If they specify a `round`, then we override the default.

Does that make sense?

Reply:
yep, same understanding.
This PR does the same, override if present, if not follow code list :+1: